openedx/features/journals/README.rst

```diff
@@ -0,0 +1,37 @@
+Journals
```

Comment:
Any reason why the `journals` app was added within edx-platform instead of in its own repo?